src/commands/cloudmanager/commerce/bin-magento/app/config/dump.js

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const BaseCommerceCliCommand = require('../../../../../../base-commerce-cli-command')
+const { getProgramId } = require('../../../../../../cloudmanager-helpers')
+const commonFlags = require('../../../../../../common-flags')
+const commonArgs = require('../../../../../../common-args')
+
+class AppConfigDumpCommand extends BaseCommerceCliCommand {
+  static strict = false
```

Comment:
This is the reason I added `babel-eslint`. This is the solution that oclif documentation provides for variable length args, but it errored out on `npm run lint-fix`. After researching it appears that eslint does not support static properties on classes out of the box. 

oclif documentation: https://oclif.io/docs/args
^ excerpt I am referring to: "For variable length arguments, disable argument validation with static strict = false on the command."

Reply:
I think you can set static on the class and it accomplishes the same thing. Used here: https://github.com/adobe/aio-cli-plugin-cloudmanager/blob/main/src/commands/cloudmanager/ip-allowlist/bind.js#L42 (amongst other places).

But... if babel-eslint is really necessary, shouldn't `@babel/eslint-parser` be used? Per https://github.com/babel/babel-eslint that's the new location.

Reply:
Thanks, you are correct. `AppConfigDumpCommand.strict = false` did the trick

Reply:
I pushed up the requested change, this is resolved